interpretation/ww_head.py

```diff
@@ -325,25 +370,6 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
             return [Mailbox().respond(msg, True)]
         help_msg += "`" + prefix + "cc` - Create a new conspiracy channel.\n"
 
-        '''donate'''
```

Comment:
The donate command was placed in te participant section, meaning that participants could give each other (or themselves) more conspiracy channels.
I moved it to the Game Master section, so that the command can only be used by Game Masters.